guillotina/tests/test_postgres.py

```diff
@@ -329,34 +331,54 @@ async def test_should_not_resolve_conflict_error_with_resolve(db, dummy_guilloti
         await cleanup(aps)
 
 
-@pytest.mark.skipif(DATABASE in ("cockroachdb", "DUMMY"), reason="Cockroach not support simple...")
-async def test_should_not_resolve_conflict_error_with_simple_strat(db, dummy_guillotina):
-    aps = await get_aps(db, "simple")
-    with TransactionManager(aps) as tm, await tm.begin() as txn:
-        ob1 = create_content()
-        ob2 = create_content()
-        txn.register(ob1)
-        txn.register(ob2)
+@pytest.mark.skipif(DATABASE in ("DUMMY",), reason="DUMMY not support simple...")
+async def test_should_raise_conflict_error_on_concurrent_insert(db, container_requester):
```

Comment:
For some reason the existing test was marked as `Cockroach not support simple...` but I think it works like in postgres